### PR TITLE
Remove reference to re.IGNORECASE

### PR DESCRIPTION
**Pull request by narrieta**
re.IGNORECASE is not available on Python 2.6

### CODE REVIEW

**Review by larohra — APPROVED** [review]

File: tests/data/hostgaplugin/ext_conf.xml [hunk 1]
```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (kevinclark19a) [inline tests/data/hostgaplugin/ext_conf.xml:1]: I'm a bit confused by the inclusion of this file-- it doesn't quite fit the PR name. Is it supposed to be here?

File: azurelinuxagent/common/protocol/wire.py [hunk 2]
```diff
@@ -852,7 +852,7 @@ def save_if_not_none(goal_state_property, file_name):
                 self._save_cache(self._goal_state.ext_conf.get_redacted_xml_text(), EXT_CONF_FILE_NAME.format(self._goal_state.incarnation))
 
             if self._extensions_goal_state is not None:
-                self._save_cache(self._extensions_goal_state.get_redacted_vm_settings(), VM_SETTINGS_FILE_NAME.format(self._extensions_goal_state.etag))  # TODO: need to redact settings
+                self._save_cache(self._extensions_goal_state.get_redacted_vm_settings(), VM_SETTINGS_FILE_NAME.format(self._extensions_goal_state.etag))
```
Comment (kevinclark19a) [inline azurelinuxagent/common/protocol/wire.py:855]: This change seems unrelated to the stated PR name. Should it belong here?

Comment (narrieta) [reply azurelinuxagent/common/protocol/wire.py:855]: when i revisited the code for get_redacted_vm_settings i realized this TODO is already done, so removing it here

File: tests/data/hostgaplugin/ext_conf.xml (hunk 1 above)
Comment (narrieta) [reply tests/data/hostgaplugin/ext_conf.xml:1]: it's used by the unit test

**Review by nagworld9 — APPROVED** [review]
